iceoryx_posh/test/integrationtests/test_posh_mepoo.cpp

```diff
@@ -299,7 +299,8 @@ class Mepoo_IntegrationTest : public Test
     }
 
     template <uint32_t size>
-    bool sendreceivesample(const int& times)
+    bool sendreceivesample(const int& times,
```

Comment:
```suggestion
    bool sendReceiveSample(const int& times,
```

Reply:
Also, the `int` could be replace with e.g. a `uint32_t` throughout the whole integration test.